This patch was 'rejected' by upstream at least for now. Drop this from next
release of UDisks 2.x:

http://bugs.freedesktop.org/show_bug.cgi?id=36361#c5

Should be replaced by the udev rules file installed by ntfs-3g package.

--- src/udiskslinuxfilesystem.c
+++ src/udiskslinuxfilesystem.c
@@ -164,6 +164,7 @@
   "vfat",
   "exfat",
   "ntfs",
+  "ntfs-3g",
   NULL,
 };
 
